test/WebSites/TagHelpersWebSite/Views/Employee/DuplicateAntiforgeryTokenRegistration.cshtml

```diff
@@ -0,0 +1,11 @@
+﻿@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+@{
+    Layout = null;
+}
+<html>
+<body>
+    @*Form tag helper already registers antiforgery token. The Html helper call should be no-op*@
+    <form asp-antiforgery="true" asp-action="DuplicateAntiforgeryTokenRegistration">@Html.AntiForgeryToken()</form>
```

Comment:
What is this test for? Should be covered completely by the first `<form>` in the other new `.cshtml` file. I'm also very surprised we don't already have `Html.AntiForgeryToken()` calls in our functional test sites.